packages/grafana-data/src/types/logs.ts

```diff
@@ -131,6 +131,11 @@ export interface DataSourceWithLogsContextSupport<TQuery extends DataQuery = Dat
    */
   getLogRowContext: (row: LogRowModel, options?: LogRowContextOptions, query?: TQuery) => Promise<DataQueryResponse>;
 
+  /**
+   * Retrieve the context query object for a given log row
+   */
+  getLogRowContextQuery?: (row: LogRowModel, options?: LogRowContextOptions, query?: TQuery) => Promise<TQuery | null>;
+
```

Comment:
I'd mention that this is currently used for showing split open button in log context

Reply:
Fixed in [03b1f89](https://github.com/grafana/grafana/pull/66777/commits/03b1f89db7ac9138776090ffe0e95d4c1cc0556d)